credentials/settings/devstack.py

```diff
@@ -45,11 +45,11 @@
 STATICFILES_STORAGE = os.environ.get('STATICFILES_STORAGE', 'django.contrib.staticfiles.storage.StaticFilesStorage')
 STATIC_URL = os.environ.get('STATIC_URL', '/static/')
 
-OAUTH2_PROVIDER_URL = 'http://edx.devstack.lms:18000/oauth2'
+OAUTH2_PROVIDER_URL = 'http://local.philanthropyu.org:18000/oauth2'
 SOCIAL_AUTH_EDX_OIDC_KEY = os.environ.get('SOCIAL_AUTH_EDX_OIDC_KEY', 'credentials-key')
 SOCIAL_AUTH_EDX_OIDC_SECRET = os.environ.get('SOCIAL_AUTH_EDX_OIDC_SECRET', 'credentials-secret')
-SOCIAL_AUTH_EDX_OIDC_ISSUER = os.environ.get('SOCIAL_AUTH_EDX_OIDC_ISSUER', 'http://edx.devstack.lms:18000/oauth2')
-SOCIAL_AUTH_EDX_OIDC_URL_ROOT = os.environ.get('SOCIAL_AUTH_EDX_OIDC_URL_ROOT', 'http://edx.devstack.lms:18000/oauth2')
+SOCIAL_AUTH_EDX_OIDC_ISSUER = os.environ.get('SOCIAL_AUTH_EDX_OIDC_ISSUER', 'http://local.philanthropyu.org:18000/oauth2')
+SOCIAL_AUTH_EDX_OIDC_URL_ROOT = os.environ.get('SOCIAL_AUTH_EDX_OIDC_URL_ROOT', 'http://local.philanthropyu.org:18000/oauth2')
```

Comment:
We can avoid this core change. By adding environment variables in devstack credentials service environment key.

Reply:
We are doing this here because we had done this same change in same file(line: 54) and other files of this repository.

Reply:
Actually `SOCIAL_AUTH_EDX_OIDC_URL_ROOT` is already defined in docker-compose file of devstack.  So this line `os.environ.get('SOCIAL_AUTH_EDX_OIDC_URL_ROOT', 'http://local.philanthropyu.org:18000/oauth2')` won't return `http://local.philanthropyu.org:18000/oauth2` but value defined in environment variable `SOCIAL_AUTH_EDX_OIDC_URL_ROOT`.. Please unit test this thing by running credentials service and checking `SOCIAL_AUTH_EDX_OIDC_URL_ROOT` value in shell, If it passed then we are good to go.

Reply:
<img width="690" alt="Screen Shot 2020-04-21 at 12 27 26 PM" src="https://user-images.githubusercontent.com/26452654/79837810-7b646c00-83cb-11ea-8c92-232b916ed568.png">

I have checked it, And as i told you it won't return **http://local.philanthropyu.org:18000/oauth2** but value in 
```
environment: # of credentials service
SOCIAL_AUTH_EDX_OIDC_URL_ROOT
```
that is 'http://edx.devstack.lms:18000/oauth2'

Reply:
@ahmed-faraz46 Should we make a change in docker-compose file for `SOCIAL_AUTH_EDX_OIDC_URL_ROOT` as @zeshanarifios suggested?

Reply:
I just rechecked it, yes I would say we should add following environment variables in our credentails service defined in `devstack/docker-compose.yml`

1. 1. SOCIAL_AUTH_EDX_OIDC_URL_ROOT (It's already defined we just need to change the value)
After removing code changes and just changing this value please test again, I guess it will work, if not then we can add following variables as well.
2. OAUTH2_PROVIDER_URL
3. SOCIAL_AUTH_EDX_OIDC_ISSUER

Reply:
This change is discarded now.